action.yml

```diff
@@ -25,6 +33,14 @@ inputs:
     description: "Install kubectl"
     default: "yes"
     required: false
+  install-vals:
+    description: "install vals"
+    default: "yes"
+    required: false
+  install-sops:
+    description: "install sops"
+    default: "yes"
```

Comment:
For those who don't need this tool, I want to set the default to `no` to shorten the installation time.The default value should be `false` so that users who do not need this tool can omit the setting and shorten the execution time.